app/routes/orders/goods.js

```diff
@@ -1,6 +1,16 @@
 import detail from "./detail";
 
 export default detail.extend({
+  showMessage: false,
+
+  beforeModel(transition) {
+    const previousRoutes = this.router.router.currentHandlerInfos;
+    const previousRoute = previousRoutes && previousRoutes.pop().name;
+    if (previousRoute === "submitted_orders") {
```

Comment:
I'd suggest that the message is passed to the page, rather than having the page check the history.
That makes it more modular.

Reply:
Removed this logic and replaced it with packages recently updated time.